### ISSUE TITLE
Grafana Variables don't work

### ISSUE THREAD

**Issue by amirfvelis** [mtanda/grafana-aws-athena-datasource#55]
I tried every possible way,
seems like i can't get any data from Athena using Grafana variables.
see attached
https://i.ibb.co/17gNjnb/Screenshot-from-2021-06-15-14-30-07.png

No value is appearing using queries:
select id from table group by 1
select 1

only All or None

Any Ideas?
Thanks for the help.

**Comment by SayaliPat** [comment]
If you have a variable as 'key' did you try providing $key in the query ?  I was able to work with that.

**Comment by robert-schmidtke** [comment]
@amirfvelis for what it's worth, I have added support for Dashboard Variables in my fork: https://github.com/robert-schmidtke/grafana-aws-athena-datasource

As long as your query returns a single column, you should be good.

![image](https://user-images.githubusercontent.com/1101741/132750054-8d28c5cd-041d-4f80-a550-4229dcbbd5bd.png)

**Comment by djesionek** [comment]
@robert-schmidtke would be really great to have that here. Have you considered making a PR with that modification?

**Comment by robert-schmidtke** [comment]
> @robert-schmidtke would be really great to have that here. Have you considered making a PR with that modification?

Yeah I have, but I won't have the time to maintain the PR, as I don't think it meets the author's coding standards. I also have reason to believe that Grafana Labs will release an official plugin themselves, as I have reached out to them about it, and they said it was on their roadmap.
